.github/workflows/package_delete_usernames.json

```diff
@@ -0,0 +1,19 @@
+{
+  "usernames": [
+    {
+      "username": "keizer619"
+    },
+    {
+      "username": "Kishanthan"
+    },
+    {
+      "username": "anupama-pathirage"
+    },
+    {
+      "username": "manuranga"
+    },
+    {
+      "username": "niveathika"
+    }
+  ]
+}
```

Comment:
@dilanSachi Lets include a new line in the end

Reply:
@keizer619 done